site/content/building-apps-for-livechat/extending-livechat-ui.md

```diff
@@ -64,10 +68,59 @@ Fullscreen apps help to reduce _context switching_. Users no longer need to swit
 
 See how to create a fullscreen app in the [documentation](https://developers.livechatinc.com/docs/agent-app-widgets/).
 
-### App settings
+#### App settings
 
 Some applications require settings configuration, especially when they use external services. After installing the app, users need to do the setup in order for the application to work.
 
 We provide a special place called Settings page, where you can embed a page with configuration. It's recommended to use it together with [Sign in with LiveChat](/docs/sign-in-with-livechat/).
 
-See how to setup a Settings page for your app in the [documentation](https://developers.livechatinc.com/docs/agent-app-widgets/).
\ No newline at end of file
+See how to setup a Settings page for your app in the [documentation](https://developers.livechatinc.com/docs/agent-app-widgets/).
+
+## Extending the Chat Widget
+
+### Mobile Chat Widgets
+Have you got a mobile app? You can still make use of LiveChat and improve your *mobile* user experience! All you need to do is install the mobile **Chat Widget** compatible with your app.
+
+- Mobile [Chat Widget for iOS](https://developers.livechatinc.com/docs/ios-widget/) apps
+- Mobile [Chat Widget for Android](https://developers.livechatinc.com/docs/android-widget/) apps
+- Mobile [Chat Widget for React Native](https://developers.livechatinc.com/docs/react-native-livechat/) apps 
+ 
+
+### Rich messages
```

Comment:
Rich messages tak naprawdę na razie jest tylko użyteczne z ChatBotami, przesunałbym to za JavaScript API i Moments. Wydaje mi się też, że JavaScript API powinno być tuż za Mobile Chat Widgetami - jest znacznie popularniejsze niż Momenty

Reply:
Chyba że jakaś inna logika stoi za tą kolejnością

Reply:
Mobile dałam na początek jako coś oczywistego i naturalnego, po prostu mobilna wersja chat widgeta. Potem Rich messages, bo wydaja mi się reprezentatywne. Potem Moment jako rodzaj RM (ja tak o tym mysle, ale moze zle). Chciałam dać API i SDK na koniec jako te bardziej zaawansowane metody.

Reply:
Moze zamienic RM z Moments? Wydaje mi się, że Mobilne powinny byc na poczatku, a JS API i Customer SDK na koncu, i ze pasuje jak te 2 ostatnie są po sobie

Reply:
Ok, rozumiem, ale podstawowe użycie JS API jest dość łatwe i dotyczy w chwili obecnej znacznie większej ilości klientów (częściej chcą dodać przycisk / ustawić name visitora, niż użyć rich messages czy momentów, szczególnie że teraz trudno ich użyć), więc nie wiem czy zamiast kolejności "od najłatwiejszej do najtrudniejszej" nie powinniśmy zrobić "od najczęściej używanych"

Reply:
dobra, przekonales mnie :D 
Mobile, JS API, Moments, RM, Customer SDK